src/main/scala/jsy/student/Lab4.scala

```diff
@@ -238,7 +238,7 @@ object Lab4 extends jsy.util.JsyApplication with Lab4Like {
     subst(e) // change this line when you implement capture-avoidance
   }
 
-  /* Check whether or not an expression is reducible given a mode. */
+  /* Check whether or not an expression is reduced enough to be applied given a mode. */
```

Comment:
I think it really should be "reducible" even if it sounds weird from a normal prose perspective.

Reply:
I guess this doesn't really matter and is probably clearer in the revision. So resolving.